source/_docs/z-wave/controllers.markdown

```diff
@@ -11,22 +11,9 @@ footer: true
 
 ## {% linkable_title Supported Z-Wave USB Sticks & Hardware Modules %}
 
-You need to have a [supported Z-Wave USB stick or module](https://github.com/OpenZWave/open-zwave/wiki/Controller-Compatibility-List) installed.
-
-| Device                    | Works on Linux | Works on Windows | Works on OSX |
-|---------------------------|----------------|------------------|--------------|
-| Aeotec Z-Stick            |   &#10003;     |                  |              |
```

Comment:
Let's keep a list of controllers that work. People want to see an easy list and we already have that info.